packages/quantification/src/__tests__/net-quantification.test.ts

```diff
@@ -264,6 +262,39 @@ describe('getNetQuantificationProjection', () => {
     ]);
   });
 
+  it('should not return a value for years that are in a other fields', () => {
```

Comment:
nit: in other fields

Reply:
double nit: that are only* in other fields

Reply:
Modified